tests/php/UsedOnTableTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SilverStripe\Admin\Tests;
+
+use SilverStripe\Admin\Forms\UsedOnTable;
+use SilverStripe\Assets\File;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Dev\TestOnly;
+
+class UsedOnTableTest extends SapphireTest implements TestOnly
+{
+    protected static $fixture_file = 'UsedOnTableTest.yml';
+
+    public function testOK()
+    {
+        $usedOnTable = UsedOnTable::create("Used On");
+
+        $file1 = $this->objFromFixture(File::class, 'file1');
+        $usedOnTable->setRecord($file1);
+
+        $request = new HTTPRequest('GET', '/');
+        $response = $usedOnTable->usage($request);
+
+        $this->assertEquals(<<<JSON
+{"usage":[{"id":"SilverStripe\Assets\Shortcodes\FileLink\/1","title":"#1","type":"File Link","state":null,"link":null}]}
```

Comment:
JSONDecoding the body and comparing it to a PHP array will probably give you more consistent results.

With a plain string, whatever you test passes depends on things like formatting and the order of keys.